packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js

```diff
@@ -289,7 +315,11 @@ export default class AgoraIpV4OrderController {
         this.user.ovhSubsidiary,
         this.catalogName,
       ).then((ipOffers) => {
-        const ipOfferDetails = ipOffers.map(this.createOfferDto.bind(this));
+        const ipOfferDetails = ipOffers
+          .filter(
+            (plan) => plan.planCode.match(/^ip-v4.*|ip-failover.*/) != null,
```

Comment:
```suggestion
            (plan) => /^ip-v4|^ip-failover/.test(plan.planCode),
```

Since we don't use the match result and to fix the sonar issue

Reply:
Done Thanks !!